vllm/model_executor/layers/linear.py

```diff
@@ -90,13 +89,9 @@ def apply(self,
               x: torch.Tensor,
               bias: Optional[torch.Tensor] = None) -> torch.Tensor:
         weight = layer.weight
-        if self.separate_bias_add:
-            if bias is not None:
-                return tgemm.mm(x, weight) + bias
-            return tgemm.mm(x, weight)
-        elif bias is not None:
-            return F.linear(x, weight, bias)
-        return tgemm.mm(x, weight)
+        if self.separate_bias_add and bias is not None:
+            return tgemm.mm(x, weight) + bias
+        return tgemm.mm(x, weight, bias)
```

Comment:
If `not separate_bias_add and bias is not None`, instead of using `F.linear(x, weight, bias)` now, there's a chance we will end up using `_custom_mm(x, weight) + bias`. Are we confident that this is faster than the fused kernel called by `torch`?